GreenhouseNode/GreenhouseNode/handlers/WindowHandler.c

```diff
@@ -0,0 +1,102 @@
+/*
+ * WindowHandler.c
+ *
+ * Created: 12/14/2021 2:27:42 PM
+ *  Author: tolde
+ */ 
+#include <ATMEGA_FreeRTOS.h>
+#include <rc_servo.h>
+#include <task.h>
+#include "WindowHandler.h"
+#include "controllers/WServo.h"
+
+EventGroupHandle_t startGroup_task;
+EventBits_t dBit;
+
+typedef struct direction{
+	int8_t direction_servo;
+	int8_t windowToDestroy;
```

Comment:
This value is never set, it is only used to delete the task? What is it supposed to do?